.appveyor.yml

```diff
@@ -30,6 +33,7 @@ install:
 build_script:
   - npm run compile
   - npm run lint
+  - npm config set scripts-prepend-node-path true
```

Comment:
Enforces all npm scripts run with the node version configured here and not the default one.